themes/digital.gov/layouts/news/card-news.html

```diff
@@ -0,0 +1,105 @@
+{{/* Display a news card for internal or external blog links.
+  External links use both source and source_url to set the logo and external link.
+*/}}
+
+{{/* Flag to display internal or external card styles */}}
+{{ $isExternal := false }}
+{{ if and (.Params.source) (.Params.source_url) }}
+  {{ $isExternal = true }}
+{{ end }}
+
+{{/* Get the source to lookup the logo for an external image */}}
+{{ $source := $.Site.GetPage "page" (print "source_" .Params.source ) }}
+{{/* Unique ID to link article to its title in screen readers */}}
+{{ $titleID := (printf "%s%s" "title-" .Title) | urlize }}
+{{ $title := .Title }}
+{{ $kicker := .Params.kicker }}
+{{ $deck := .Params.deck }}
+{{ $summary := .Params.summary }}
+{{ $link := .Permalink }}
+{{ $short_url := .Params.short_url }}
+{{ $source_url := .Params.source_url }}
+
+
+<article
+  class="card {{ if $isExternal }}
+    card-news--external
+  {{ else }}
+    card-news--internal
+  {{ end }}card-linked"
+  {{ if $short_url }}data-short_url="{{- $short_url -}}"{{ end }}
+  aria-labelledby="{{ $titleID }}"
+>
+  <div class="card__content">
+    {{ if $isExternal }}
+      <div class="card__summary">
+        <h3 id="{{ $titleID }}">
+          <a href="{{- $link -}}">{{- $title | markdownify -}}</a>
+        </h3>
+        <p>
+          {{- if $deck -}}
+            {{- $deck | markdownify | emojify -}}
+          {{- end -}}
+          {{- if $source -}}
+            {{ $url := urls.Parse $source_url }}
+            <em class="card__source source"
+              >— via
+              <a
+                href="{{- if $short_url -}}
+                  {{- $short_url -}}
+                {{- else -}}
+                  {{- $source_url -}}
+                {{- end -}}"
```

Comment:
You could improve readability and maintainability here if you moved this to a variable you call later.

Similar to what you did with the `$isExternal` variable.